### PR TITLE
chore(RO): change RO fields order

### PR DESCRIPTION
**Pull request by johnytiago**
Ticket: https://trello.com/c/ef7Mw780/1237-reorder-cms-fields-for-research-outputs

- feat(CI): apply schema to PR squidex app
- chore(RO): change RO fields order

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -156,6 +156,8 @@ deploy:sls:branch:
     - ./ci/config-squidex-rules.sh ./backup/rules
     # create-app.py configures sq to new app
     - sq sync in backup
+    # apply schema changes on top of dev backup
+    - sq sync in packages/squidex/schema -t schemas --delete
```
Comment (peterstarling) [inline .gitlab-ci.yml:160]: hm.... i dont mind adding it to the PR but... do you think maybe this should run on every commit on a non master branch? Imagine a scenario:

1. An engineer raises a branch with some code changes (not schema)
2. That branch is deployed, dev backup restored and no other changes applied (as per the change you are proposing)
3. That engineer than decides to make changes to the schema and pushes a commit
4. No changes are applied as this jobs trigger is a new PR only (right?)

File: packages/squidex/schema/schemas/research-outputs.json [hunk 2]
```diff
@@ -60,27 +74,94 @@
           "placeholder": "",
           "isRequired": true,
           "isRequiredOnPublish": false,
-          "isHalfWidth": false
+          "isHalfWidth": true
```
Comment (peterstarling) [inline packages/squidex/schema/schemas/research-outputs.json:77]: this intended? (nothing in the ticket description which is why i ask)

Comment (peterstarling) [inline packages/squidex/schema/schemas/research-outputs.json:77]: this intended?

**Review by peterstarling — CHANGES_REQUESTED** [review]
The diff is a little difficult to follow (i dont remember how to turn on a more intelligent diff and whether it would help here) so it's worth looking over the changes you proposed (since they may have been generated by syncing the schema with sq cli).

File: .gitlab-ci.yml (hunk 1 above)
Comment (johnytiago) [reply .gitlab-ci.yml:160]: This should do just that.
We have this verification step, but every commit inside a PR will pass it.
https://github.com/yldio/asap-hub/pull/876/files/acb2c10995ae13dc5b197168a309436e79997011#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7L14

File: packages/squidex/schema/schemas/research-outputs.json (hunk 2 above)
Comment (johnytiago) [reply packages/squidex/schema/schemas/research-outputs.json:77]: Yeah. This makes a field only take half of the page on the CMS

Comment (johnytiago) [reply packages/squidex/schema/schemas/research-outputs.json:77]: Yeah. This makes a field only take half of the page on the CMS

**Review by peterstarling — APPROVED** [review]

**Pull request merged**